packages/stacked/README.md

```diff
@@ -458,14 +458,14 @@ class InformationService with ReactiveServiceMixin { //1
 }
 ```
 
-Easy peasy. This service can now be listened too when any of the properties passed into the `listenToReactiveValues` is changed. So how do listen to these values? I'm glad you asked. Lets move onto the `ReactiveViewModel`.
+Easy peasy. This service can now listen too when any of the properties passed into the `listenToReactiveValues` is changed. So how do you listen to these values? I'm glad you asked. Let's move onto the `ReactiveViewModel`.
```

Comment:
add back "be listened" to. The service is not listening to itself. The ViewModel is listening to it.